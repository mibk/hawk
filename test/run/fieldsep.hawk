BEGIN {
	FS = ";+"
	OFS = "."
}

{
	print $1, $2, $3, $4
}
