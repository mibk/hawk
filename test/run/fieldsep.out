 one.two.three.  four
next.-.end.
